packages/asc-core/src/styled-components.ts

```diff
@@ -1,6 +1,14 @@
 import * as styledComponents from 'styled-components'
 import { Theme } from './theme'
 
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      styledComponent: any
```

Comment:
what does this do and why is styledComponents: any?

Reply:
This ensures that a styled component type sent as parameter can be rendered as <styeldComponent></styledComponent>. I changed the way the exported style was rendered so this can be removed.